tests/serverpod_test_server/test_integration/database_operations/transactions/transaction_isolatation_level_test.dart

```diff
@@ -0,0 +1,292 @@
+import 'dart:async';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  var session = await IntegrationTestServer().session();
+
+  tearDown(() async {
+    await SimpleData.db.deleteWhere(session, where: (t) => Constant.bool(true));
+  });
+
+  group(
+      'Given read committed transaction isolation level and single row in database',
+      () {
+    var settings = TransactionSettings(
+      isolationLevel: IsolationLevel.readCommitted,
+    );
+
+    test(
+        'when row is modified after first statement in transaction'
```

Comment:
nit: when breaking test descriptions up like this (which I really like) then a space has to be added to the end of each line otherwise the resulting text is concatenated.

Reply:
Aah thanks! I stole this from you I think, i also really like it! Will fix 🙏

Reply:
787086293005dec16c4d270ae70baf326e0a69da